src/EditorFeatures/CSharpTest/InvertIf/InvertIfTests.Elseless.cs

```diff
@@ -837,5 +838,110 @@ public string Prop
     }}
 }}");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        [WorkItem(51359, "https://github.com/dotnet/roslyn/issues/51359")]
+        public async Task TestIsCheck_CSharp8()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    int M()
+    {
+        [||]if (c is object)
+        {
+            return 1;
+        }
+        else
+        {
+            return 2;
+        }
+    }
+}",
+@"class C
+{
+    int M()
+    {
+        if (!(c is object))
```

Comment:
💭 Even in C# 8 we can use `if (c is null)` here

Reply:
done.